src/shared/vlan-util.c

```diff
@@ -17,9 +17,10 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
-#include "vlan-util.h"
-#include "parse-util.h"
+#include "basic/string-util.h"
```

Comment:
"basic" is part of the search path anyway, for everything we build from our search tree. Please drop this prefix